apps/docs/src/app/platform/component-docs/platform-forms/platform-input/platform-input-example/platform-input-auto-complete-validation-example.component.html

```diff
@@ -21,13 +15,9 @@
     <div *ngIf="!itemTemplate" class="fd-popover__body fd-popover__body--no-arrow">
```

Comment:
Not sure why but `<div class="fd-popover__body fd-popover__body--no-arrow">` wrapper brings incorrect popover positioning (extra shift from left side).

![image](https://user-images.githubusercontent.com/3263744/93223207-79cf8200-f778-11ea-9508-619c33d59dcc.png)

Pleasere remove this wrapper.

Also could you please rid of `ng-template` and put popover template directly into `fd-popover-body` to make it more readable.

Reply:
@dimamarksman I removed the wrapper, but the popover was still not positioned correctly. So I had to force an popover update in the example controller code.

I also needed to force a popover update, because when the popover is positioned above the input, and the list is shortened via filter, the popover wasn't repositioning and it became "detached" from the input field.